CONTRIBUTING.md

```diff
@@ -125,34 +125,44 @@ Go to <http://localhost:6006> for the widget playground.
-    ▸ .storybook/         << The storybook configuration
-    ▸ stories/            << The widget stories
-    ▸ test/               << Utilities for the tests
-  ▸ mocks/                << Fake versions of the API, for testing
-  ▸ testutils/            << Global utilities for the tests
-▸ website/                << The generated website
-▸ scripts/                << The scripts for maintaining the project
-  CHANGELOG.md            << The autogenerated changelog (based on commits)
-  CONTRIBUTING.md         << this file
-  package.json            << The definition of the project
-  README.md               << The introduction of the project
+▸ examples/                            << Examples, grouped per flavor
+▸ packages/                            << Packages of the project
+  ▸ react-instantsearch/               << Bundled React InstantSearch library
+  ▸ react-instantsearch-core/          << Runtime-independent React InstantSearch version
+  ▸ react-instantsearch-dom/           << DOM-specific React InstantSearch version
+  ▸ react-instantsearch-native/        << React Native-specific InstantSearch version
+  ▸ react-instantsearch-dom-maps/      << DOM-specific React InstantSearch version with Google Maps
+
+  ▸ react-instantsearch-hooks/        << React InstantSearch Hooks library
+  ▸ react-instantsearch-hooks-web/    << DOM-specific React InstantSearch Hooks version
+  ▸ react-instantsearch-hooks-server/ << Server-specific React InstantSearch Hooks version
+
+  ▸ instantsearch.js/                  << The InstantSearch.js library
+▸ tests/                               << The test utilites
+  ▸ mocks/                             << Fake versions of the API, for testing
+  ▸ utils/                             << Global utilities for the tests
+▸ website/                             << The generated website
+▸ scripts/                             << The scripts for maintaining the project
+  CHANGELOG.md                         << The autogenerated changelog (based on commits)
+  CONTRIBUTING.md                      << this file
+  package.json                         << The definition of the project
+  README.md                            << The introduction of the project
 ```
 
 ### InstantSearch.js source folder
 
 ```
-▸ src/
+▸ src/                             << The source of the library
   ▸ components/       << The Preact components for the UI of our widgets
   ▸ connectors/       << The source of all the connectors driving the widgets' logic
   ▸ helpers/          << The source of the method helpers
   ▸ lib/              << The core of the library (InstantSearch, routers, etc.)
   ▸ middleware/       << The source of the middleware
   ▸ types/            << The TypeScript declarations
   ▸ widgets/          << The source of the widgets
+▸ .storybook/                      << The storybook configuration
+▸ stories/                         << The widget stories
+▸ test/                            << Utilities for the tests
 ```
```

Comment:
Should this remain? We probably won't have it for all packages.

Reply:
It's equal for me, i guess it doesn't bring much value, but on the other hand, we want to make it as approachable as possible?

Reply:
If we keep it, then let's add it for React InstantSearch packages?